modules/tinymce/src/themes/silver/main/ts/ui/window/SilverDialogInstanceApi.ts

```diff
@@ -124,12 +126,13 @@ const getDialogApi = <T extends Dialog.DialogData>(
 
   const redial = (d: Dialog.DialogSpec<T>): void => {
     withRoot((root) => {
+      const id = access.getId();
```

Comment:
I didn't really mean to leave this in but I think it's worth keeping anyone. Anyways, it was added to ensure that redial only works on the dialog it's called on and not others if multiple were open (this was just something I noticed could have been an issue while playing).